test/models/customer_test.rb

```diff
@@ -0,0 +1,112 @@
+require "test_helper"
+
+describe Customer do
+  describe "relationships" do 
+    it "has movies" do 
+      # Arrange
+      mariya = customers(:mariya)
+      up = movies(:up)
+      avengers = movies(:avengers)
+      zootopia = movies(:zootopia)
+      
+      #Act
```

Comment:
It seems you don't use the variable `rental`. As such, you don't need this assignment.